pkgs/games/kobodeluxe/default.nix

```diff
@@ -9,11 +9,22 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ SDL SDL_image libGLU libGL ];
+  nativeBuildInputs = [ makeWrapper ];
 
   prePatch = ''
     sed -e 's/char \*tok/const char \*tok/' -i graphics/window.cpp
   '';
 
+  preConfigure = ''
+    substituteInPlace ./configure --replace "CONFIGDIR='HOME>>'" "CONFIGDIR='HOME>>/.config/${pname}/'";
+  '';
+
```

Comment:
```Suggestion
```

Adding the scores flag should work around the issue already, will it not?

Reply:
Yes, I but using a XDG dir for the configuration is nicer than dumping the files in ~/ 
The scores directory is separate from the config directory.

Reply:
Problem is that that's not XDG compliant either as it doesn't read any env vars. If upstream doesn't care about XDG, we usually don't patch that out. See i.e. openssh. That'd be an opinionated patch and we'd at most offer that as an opt-in flag.

Reply:
Makes sense. In this case I'll just remove this since being properly XDG compliant would require changing the source.